kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -412,7 +422,7 @@ define([
 
                     }
 
-                    $('td:eq(0)', nRow).find('i[data-caret]').on('click', e => {
+                    $('td:eq(1)', nRow).find('i[data-caret]').on('click', e => {
```

Comment:
ZOMG! This could also use `$caret`!

Reply:
oof it's good you are asking to fix this as i found a bug with the file info display i had missed otherwise 🙈